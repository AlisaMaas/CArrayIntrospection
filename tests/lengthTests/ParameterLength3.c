/*
* Tests a loop where one argument is length and
* one is an element to find. The length of x 
* should be y. 
*
* Example contributed by Peter Ohmann.
*/
int find(int* x, int y, int me){
  int i;
  for(i = 0; i < y; i++){
    if(x[i] == me)
      return i;
  }
  return -1;
}