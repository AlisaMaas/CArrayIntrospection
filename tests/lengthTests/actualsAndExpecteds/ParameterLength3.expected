Analyzing find
	Argument x has max index argument y
	Argument y has unknown max index.
	Argument me has unknown max index.
